### PR TITLE
KOGITO-6427, KOGITO-7372 : Improvements on Generics

### PR DESCRIPTION
**Pull request by inodeman**
Hi @hasys @domhanak can you review this PR? It has the improvements of both tickets for Generics

### CODE REVIEW

**Review by domhanak — APPROVED** [review]
I was not able to reproduce the issues anymore. Tests are passing, looks good to me.

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/util/StringUtilsTest.java [hunk 1]
```diff
@@ -78,6 +78,15 @@ public void testIsGenericsFormatOk() {
         assertTrue("Should be ok generics format", StringUtils.isOkWithGenericsFormat("List"));
     }
 
+    @Test
+    public void testIsGenericsFormatWrong() {
+        //wrong
+        assertFalse("Should be ok generics format", StringUtils.isOkWithGenericsFormat("<<X>>"));
```
Comment (domhanak) [inline packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/util/StringUtilsTest.java:84]: I guess the string should be `Should not be OK with generics format` Am I understanding it correctly?

**Review by hasys — APPROVED** [review]
Hello @inodeman, works good. Thank you!

**Comment by hasys** [comment]
@kiegroup/gatekeepers hello, this PR is ready for merge. Thank you!

File: packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/util/StringUtilsTest.java (hunk 1 above)
Comment (inodeman) [reply packages/stunner-editors/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/forms/util/StringUtilsTest.java:84]: Hi @domhanak yes, that string should not be ok, that is why the message if wrong, "Should be ok with generics"

**Comment by mareknovotny** [comment]
does this need to be backported to kie-wb-common repository too? @inodeman @hasys @domhanak

**Comment by domhanak** [comment]
@mareknovotny Afaik this is related to Kie Tools only so no need to backport

**Review by yesamer — APPROVED** [review]

**Pull request merged**